datacapturing/src/main/java/de/cyface/datacapturing/DataCapturingService.java

```diff
@@ -30,13 +37,25 @@
  * <p>
  * An object of this class is not thread safe and should only be used once per application. You may start and stop the
  * service as often as you like and reuse the object.
+ * <p>
+ * If your app is suspended or shutdown, the service will continue running in the background. However you need to use
+ * disconnect and reconnect as part of the <code>onStop</code> and the <code>onResume</code> method of your
+ * <code>Activity</code> lifecycle.
  *
  * @author Klemens Muthmann
  * @version 1.0.0
  * @since 1.0.0
  */
 public final class DataCapturingService {
 
+    private static final String TAG = "de.cyface.capturing";
+    private final static String ACCOUNT = "default_account";
+    private final static String ACCOUNT_TYPE = "de.cyface";
+    private final static String AUTHORITY = BuildConfig.provider;
+    public static final long SECONDS_PER_MINUTE = 60L;
+    public static final long SYNC_INTERVAL_IN_MINUTES = 60L;
+    public static final long SYNC_INTERVAL = SYNC_INTERVAL_IN_MINUTES * SECONDS_PER_MINUTE;
```

Comment:
Any reason why you choose 60 minutes as sync interval? You might wanna write it down here ...

Reply:
Comes from the example code. There is no requirement on how to set this. So I just used that value.